lib/koans/07_pattern_matching.ex

```diff
@@ -5,13 +5,18 @@ defmodule PatternMatching do
     assert match?(1, ___)
   end
 
-  koan "A pattern can change" do
+  koan "A value can be bound to a variable" do
+    a = 1
+    assert a == ___
+  end
+
+  koan "A variable can be rebound" do
```

Comment:
I felt like it was more accurate to use the language "bound" in this case. However, it seems it needed it's own introduction before talking about "rebound".